LPS-109940 Removing fields group when moving.

oliveiraaraujo opened the pull request.

lawrence-lee committed:
LPS-110421 Refactor tests to take place in workspace

lawrence-lee committed:
LPS-110421 Split tests into archetypes

AliciaGarciaGarcia committed:
LPS-111116 Show not found error inside the same site's default page

lawrence-lee committed:
LPS-110421 SF

lawrence-lee committed:
LPS-110421 Remove versions from build.gradle

lawrence-lee committed:
LPS-110421 Remove versions from pom.xml

lawrence-lee committed:
LPS-110421 Add dependency management to maven workspaces

lawrence-lee committed:
LPS-110421 Fix maven workspace testing

lawrence-lee committed:
LPS-110421 No longer needed

lawrence-lee committed:
LPS-110421 Temporarily ignore form field template checks

lawrence-lee committed:
LPS-110421 Fix Tests

lawrence-lee committed:
LPS-110421 Remove unused properties from requiredProperties

lawrence-lee committed:
LPS-110421 Extra dependency

lawrence-lee committed:
LPS-110421 For javax-servlet 3.0.1 for 7.1 and 7.2 projects

lawrence-lee committed:
LPS-110421 EXT projects not supported in maven

lawrence-lee committed:
LPS-110421 Fix allprojects bug in test

lawrence-lee committed:
LPS-110421 Test Fixes

carolmariaabb committed:
LPS-88241 Clone field without reference

carolmariaabb committed:
LPS-88241 Regen snapshots

lawrence-lee committed:
LPS-110421 Refactor tests

lawrence-lee committed:
LPS-110421 Revert spring-mvc-portlet changes

lawrence-lee committed:
LPS-110421 Fix service builder test errors

lawrence-lee committed:
LPS-110421 Apply 7.0 specific versions for theme

lawrence-lee committed:
LPS-110421 SF

lawrence-lee committed:
LPS-110421 Needs to be in workspace

lawrence-lee committed:
LPS-110421 Remove dependency management option

lawrence-lee committed:
LPS-110421 Add css builder to spring-mvc projects

lawrence-lee committed:
LPS-110421 Dependency management for spring-mvc portlets

lawrence-lee committed:
LPS-110421 Test fix for CI

lawrence-lee committed:
LPS-110421 Fix errors from test

lawrence-lee committed:
LPS-110421 Skip maven build of nested path test

lawrence-lee committed:
LPS-110421 Simplify

lawrence-lee committed:
LPS-110421 Only configure in test

lawrence-lee committed:
LPS-110421 Fix NPE

lawrence-lee committed:
LPS-110421 liferay.bom.version is more clear

lawrence-lee committed:
LPS-110421 Refactor tests

lawrence-lee committed:
LPS-110421 Remove versions

lawrence-lee committed:
LPS-110421 SF

lawrence-lee committed:
LPS-110421 Fix regex

lawrence-lee committed:
LPS-110421 Manage maven plugins at project level instead

lawrence-lee committed:
LPS-110421 Refactor remaining tests

lawrence-lee committed:
LPS-110421 Apply node plugin in test

lawrence-lee committed:
LPS-110421 SF

lawrence-lee committed:
LPS-110421 Remove unused test code

lawrence-lee committed:
LPS-110421 Refactor DTD tests

lawrence-lee committed:
LPS-110421 Use variables when possible

lawrence-lee committed:
LPS-110421 Add buildscript for CI customization

lawrence-lee committed:
LPS-110421 Force com.liferay.frontend.js.loader.modules.extender.api:1.0.2 for 7.0 projects

lawrence-lee committed:
LPS-110421 Fix rebase error

lawrence-lee committed:
LPS-110421 Enable TP by default

lawrence-lee committed:
LPS-110421 Review changes

pei-jung committed:
LPS-111509 Rename default account roles

pei-jung committed:
LPS-111509 Add descriptions for account roles

pei-jung committed:
LPS-111509 Baseline

pei-jung committed:
LPS-111509 Add upgrade for role names

lawrence-lee committed:
LPS-110421 Review changes

ricardocousolr committed:
LPS-111962 When strategy is not mirror there could still be an imported folder from a repository

oliveiraaraujo committed:
LPS-109940 Removing fields group when moving.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

lawrence-lee committed:
LPS-110421 Update to 7.3.1

lawrence-lee committed:
LPS-110421 SF

thektan committed:
LPS-111684 portal-search-admin-web: Improve code readability

https://issues.liferay.com/browse/LPS-111684

thektan committed:
LPS-111684 portal-search-admin-web: Start timer after fetch responds

https://issues.liferay.com/browse/LPS-111684

lawrence-lee committed:
LPS-110421 Test fixes for 7.3.1

lawrence-lee committed:
LPS-110421 Update project template tokens

lawrence-lee committed:
LPS-110421 maven plugin descriptors

lawrence-lee committed:
LPS-110421 Use latest portal-tools-bundle-support

lawrence-lee committed:
LPS-110421 Use latest gradle-plugins-workspace

victorg1991 committed:
LPS-110138 Use LayoutPageTemplateEntry status when it's not a widget page template

ling-alan-huang committed:
LPS-84119 Remove unnecessary backslash

ling-alan-huang committed:
LPS-84119 Auto SF

gaborlovasdotcom committed:
LRQA-57134 Add a new test to check refresh token's lifetime variable operates as expected

LRQA-57134 Add a new test to check refresh token's lifetime variable operates as expected

4lejandrito committed:
LPS-111116 Using the default status markup

4lejandrito committed:
LPS-111116 Not needed?

4lejandrito committed:
LPS-111116 Simplifying

4lejandrito committed:
LPS-111116 Fixing tests with new parameter

4lejandrito committed:
LPS-111116 Adding test

javiergamarra committed:
LPS-112013 regenerate

david-truong committed:
LPS-112052 set app.server.tomcat.version automatically

david-truong committed:
LPS-112052 gradleTest

thektan committed:
LPS-111684 portal-search-admin-web: Improve maintanability

- Use namespace to avoid chances of getting wrong node
- Use the same selector for checking in progress nodes
- Move interval change logic closer to where used

https://issues.liferay.com/browse/LPS-111684

Commit:
LRQA-57946 - Enabling Workflow acceptance passing tests (remove out of quarantine)

david-truong committed:
LPS-112052 fix gradleTest, update to gulp 4

ling-alan-huang committed:
LPS-84119 Add testcases

ling-alan-huang committed:
LPS-84119 Enforce naming variables of type DoubleStream, IntStream and LongStream

ling-alan-huang committed:
LPS-84119 SF, fix violations

JoyceWang08211 committed:
LRQA-57909 Refine relevant scope for export-import module

JoyceWang08211 committed:
LRQA-57913 Remove useless test

JoyceWang08211 committed:
LRQA-57914 Refine relevant scope for staging module

carolmariaabb committed:
LPS-111775 Add language keys

brianwulbern committed:
LRQA-51488 Fix typo

markocikos committed:
LPS-110926 Extend openModal API to support setting body HTML

Commit:
LPS-111924 unit tests added

petershin committed:
LPS-85765 Update Export-Package instruction for portal-impl

Commit:
LPS-111924 SF

petershin committed:
LPS-85765 Add packageinfo files

petershin committed:
LPS-112052 Remove System.out

petershin committed:
LPS-112052 Use existing method

petershin committed:
LPS-112052 Move task creation logic

petershin committed:
LPS-112052 Avoid using def keyword

petershin committed:
LPS-112052 Source formatting

carolmariaabb committed:
LPS-111775 Implement Header

Commit:
LPS-111924 This test does not make sense as we are doing a mapping so returned form must be equals to the expected form

Commit:
LPS-111924 Not needed, testToDDMFormWithEmptyDataDefinition is asserting all the default values

Commit:
LPS-111924 Not needed

Commit:
LPS-111775 Move parent div to inside jsp

Commit:
LPS-111924 Improve the test

ealonso committed:
LPS-111936 Set only if we have dropZones

drewbrokke committed:
LPS-111509 account-service - refactor - uses logical order: oldName > newName

drewbrokke committed:
LPS-111509 account-service - SF - collapse newlines since order of execution doesn't matter

drewbrokke committed:
LPS-111509 account-service - refactor - since the description maps don't change, we can just create them once and reuse them

drewbrokke committed:
LPS-111509 account-service - SF - inline since it's only used once

drewbrokke committed:
LPS-111509 account-service - removes title map to maintain consistency with other system roles

Commit:
LPS-111924 Add dependency

ling-alan-huang committed:
LPS-84119 Temporarily disable testcases, Alan will revert when everthing is fixed.

Commit:
LRQA-57973 Modify locator for text input field with label

ling-alan-huang committed:
LPS-84119 Auto SF

ling-alan-huang committed:
LPS-84119 SF, use MapBuilderCheck

ling-alan-huang committed:
LPS-107076 Add whitelist for control tables that do not require column companyId

dellawang committed:
LRQA-58006 Add property to avoid case run on instance

brianchandotcom committed:
LPS-112101 LPS-112013 prep next

brianchandotcom committed:
LPS-112101 LPS-112013 prep next

brianchandotcom committed:
LRQA-51488 regen

Commit:
LRQA-58004 Modify macros to type in thread reply field

Commit:
LRQA-58004 Remove abandoned locator

Commit:
LRQA-58004 Modify cases to add category and thread reply

Commit:
LRQA-58004 Remove duplicate case

AliciaGarciaGarcia committed:
LPS-111116 setting friendlyURL to null if error

victorg1991 committed:
LPS-112140 Update editables only if we are not editing, otherwise we are causing two editor creation

Commit:
LRQA-58012 Modify Xpath of RSS link in AP

pavel-savinov committed:
LPS-112147 Add more bottom spacing for the Hello World header

pavel-savinov committed:
LPS-112147 Line break between the name and the version of the build as it shows in the Hello World portlet

JoyceWang08211 committed:
LRQA-58003 Quarantine failing Site Templates tests for acceptance run

Commit:
LRQA-57999 Modify case to assert toggle controls status in widget page

liulukiwi committed:
LRQA-55437 Add locator for asset title in AP and modify macro to use it

JoyceWang08211 committed:
LRQA-57909 Fix for SF issue

liulukiwi committed:
LRQA-55437 Remove ignore for this case

Commit:
LRQA-57999 Modify case to add a widget page

Jeremy-Ch committed:
LRQA-58000 Add steps to input the content

Commit:
LRQA-57999 Add a widget page and Use JSONWS to reduce running time

victorg1991 committed:
LPS-112150 Set the margin of the control menu buttons to 8px

Commit:
LRQA-57999 Rerun case on relevant

pavel-savinov committed:
LPS-112137 Make sure that portlet used in the contributed fragments already exists when registering corresponding collection contributor

Commit:
LPS-101628 Deprecate Liferay.Util.getWindowWidth

As discussed in https://github.com/wincent/liferay-portal/pull/206,
we're marking `Liferay.Util.getWindowWidth` as deprecated

Sharryxue committed:
LRQA-58018 Use a different locator

markocikos committed:
LPS-110926 Replace Liferay.SignInModal usage with openModal utility

markocikos committed:
LPS-110926 Deprecate Liferay.SignInModal plugin

Jeremy-Ch committed:
LRQA-58017 Use a suitable way to input content

pavel-savinov committed:
LPS-112020 No need to call links service if asset entry doesn't exist

pavel-savinov committed:
LPS-112020 Move common methods to the base class

Hanlf committed:
LRQA-57878 Remove quarantine

Commit:
LRQA-58022 Rerun cases on upstream

Jeremy-Ch committed:
LRQA-57998 Update test to use the suitable path

Jeremy-Ch committed:
LRQA-58013 Remove the redundant step

pavel-savinov committed:
LPS-112020 Implement highest rated assets info list provider

Sharryxue committed:
LRQA-58023 Add a page for testing

pavel-savinov committed:
LPS-112020 Language key

pavel-savinov committed:
LPS-112020 Buildlang

javiergamarra committed:
LPS-103446 do not reuse stream

Commit:
LPS-111775 Remove unused tab

Commit:
LPS-111775 Fix language keys

Commit:
LPS-111775 buildLang

4lejandrito committed:
LPS-111408 (un)ignoring multiple redirect not found entries

liferay-translation committed:
LPS-77699 Update Translations

Commit:
LPS-111775 Rename frontend from summary to report since it will be called Report in the backend

adolfopa committed:
LPS-112157 Enable/disable reply button

Commit:
LPS-111775 Remove annoying alert

Commit:
LPS-111775 buildLang

4lejandrito committed:
LPS-111408 SF

markocikos committed:
LPS-110926 Extend tests

markocikos committed:
LPS-110926 Add a console warning if both 'url' and 'bodyHTML' params are used

Commit:
LPS-101620 Deprecate `Liferay.Util.randomInt`

The goal of these changes is to deprectate the `Liferay.Util.randomInt`
function and update the usages.

Commit:
LPS-111775 Simplify css and html structure

adolfopa committed:
LPS-112145 There may be other users in the company

When checking the user count, use the actual company count minus one (we need to
exclude the default user). In some scenarios (e.g. local execution) there may be
more users in the company.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/analytics/.gitrepo'.

brianchandotcom committed:
LPS-110421 Lawrence, not lawrence.

brianchandotcom committed:
LPS-110421 Remove period for consistency with ProjectGenerator.java

brianchandotcom committed:
LPS-110421 prep next

brianchandotcom committed:
LPS-110421 prep next

brianchandotcom committed:
LPS-110421 prep next

AliciaGarciaGarcia committed:
LPS-111116 add style to error

Commit:
LPS-101620 Update the deprecation comment for `Liferay.Util.randomInt`

As discussed in https://github.com/wincent/liferay-portal/pull/215
we think it's better to avoid proposing a replacement for this function
as it's probably application dependent. Usages sites have been updated
to use `Number.MAX_SAFE_INTEGER` instead of `Date.now()`.

adolfopa committed:
LPS-112061 Revert the BMP plugin back to 3.4.3

There are some bugs in the new 3.5 plugin that make some of our tests fail.
Revert back so that BMP processing works as before.

Commit:
LRQA-57134 Remove redundant test steps

Commit:
LRQA-57134 Define vars before usage

cgoncas committed:
LPS-112179 Send error to frontend when an error has been found.

cgoncas committed:
LPS-112179 Test it.

brianchandotcom committed:
LPS-111408 Hugo SF

brianchandotcom committed:
LPS-111408 as used in service.xml

kenjiheigel committed:
LRCI-1232 Set test.class at the top of the batch

kenjiheigel committed:
LRCI-1232 Make the match more clear

brianchandotcom committed:
LPS-101628 Hugo SF

brianchandotcom committed:
LPS-111775 Our usual order

brianchandotcom committed:
LPS-111684 Only make it a sentence if it's 2 or more sentences.

petershin committed:
LPS-112052 Semver gradle-plugins-workspace

petershin committed:
LPS-112052 Use latest gradle-plugins-workspace

joshchong committed:
LRQA-58037 Remove printing Solr logs

brookedalton committed:
LRQA-58019 Use PageEditor macro to add widget to content page

brookedalton committed:
LRQA-58019 Add and edit widget in content page

brianchandotcom committed:
LPS-112052 rename

brianchandotcom committed:
LPS-112052 prep next

brianchandotcom committed:
LPS-85765 deprecate

Jeremy-Ch committed:
LRQA-57997 Add macro for going to item selector via CKEditor

Jeremy-Ch committed:
LRQA-57997 Add macro for viewing content image

Jeremy-Ch committed:
LRQA-57997 Add path for the tool bar image button

Jeremy-Ch committed:
LRQA-57997 Update macros to use CKEditor

Jeremy-Ch committed:
LRQA-57997 Update tests for adding image via CKEditor

Commit:
LRQA-57931 Skip portal instance for tests

Commit:
LRQA-57489 Rerun cases on relevant

pavel-savinov committed:
LPS-111915 Get article display directly from the local service for the preview purposes

liuxingye committed:
LRQA-57997 Use assertVisible

dellawang committed:
LRQA-58052 Change to 'Search' due to 'Home' not widget page

jkappler committed:
LPS-111478 Pagination could be null, so control it

jkappler committed:
LPS-111478 Adds integration test for collectionselector

dellawang committed:
LRQA-57743 Update locator for AlloyEditor#EDITOR

sandrodw3 committed:
LPS-112152 Decrease z-indexes by one, so that sidebar is under control menu container

pavel-savinov committed:
LPS-111915 Ignore failing unrelated test

pavel-savinov committed:
LPS-112220 Process root page element correctly for the page definition preview purposes

Commit:
LPS-101620 Use correct format for JSDoc comment

Updated `@deprecated` comment based on
[https://github.com/brianchandotcom/liferay-portal/commit/dea258f](brianchandotcom@dea258f)

4lejandrito committed:
LPS-112141 DB2 returns rows in a different order

Service xml defines a sorting order by hits. Both entries here have the
same number of hits so the test should not rely on the relative order
with the same number of hits.

pavel-savinov committed:
LPS-112205 Move to different package to avoid packages split error

jkappler committed:
LPS-112243 Get the first collection by name based on locale, since it can change in the UI

javiergamarra committed:
LPS-103446 rename according to Pablo’s suggestions

Commit:
LPS-111924 Fix test

Commit:
LPS-101620 Fix JSDoc comment

Based on
https://github.com/wincent/liferay-portal/pull/215#discussion_r412112544

4lejandrito committed:
LPS-112245 Adding tests

boton committed:
LPS-112041 Force integer numbers in RatingsLike

boton committed:
LPS-112041 Force integer numbers in RatingsThumbs

liferay-translation committed:
LPS-77699 Update Translations

Jeremy-Ch committed:
LRQA-58008 Use a suitable way to input

Jeremy-Ch committed:
LRQA-58016 Add a new widget page for test

Jeremy-Ch committed:
LRQA-58016 Ensure navigate to the private page

vicnate5 committed:
LPS-112046 Fix compile

dgarciasarai committed:
LPS-112048 Convert to a list and fix style

brianchandotcom committed:
LPS-110926 SF

brianchandotcom committed:
LPS-112020 regen

Commit:
LPS-110613 Use a model listener to keep track of all new DDMStructures and use DeleteAfterTestRun annotation to remove them

vicnate5 committed:
LRQA-58072 Add project template unit tests to Stable

dgarciasarai committed:
LPS-112048 Use ClayList component instead of raw list

Commit:
LPS-110613 Enable/Fix tests

rafaprax committed:
LPS-107005 Add missing reindex call

brookedalton committed:
LRQA-58077 Unquarantine from relevant

yichenroy committed:
LRCI-1086 Allow local file pattern

brianchandotcom committed:
LPS-111924 SF

brianchandotcom committed:
LPS-111924 Inline

pei-jung committed:
LPS-111509 Wait until the upgrade has been completed

brianchandotcom committed:
LPS-111924 SF

Commit:
LRQA-57448 Rerun case on relevant

brianchandotcom committed:
LPS-111509 consistency

brianchandotcom committed:
LPS-111509 Same name is fine

brianchandotcom committed:
LPS-111509 Rename

oliveiraaraujo commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-7.liferay.com/job/test-portal-source-format/2475/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-7.liferay.com//userContent/jobs/test-portal-source-format/builds/2475/jenkins-report.html">here</a>.</b></p>

oliveiraaraujo commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest(master)/5707/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-4.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5707/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 2 minutes 55 seconds 528 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/06fc842158775ba2b96fcd2e9740b8cbd5137de1">06fc842158775ba2b96fcd2e9740b8cbd5137de1</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/oliveiraaraujo/liferay-portal/tree/LPS-109940">LPS-109940</a><br/>Branch GIT ID: <a href="https://github.com/oliveiraaraujo/liferay-portal/commit/abd9ffc838dbd33170ab0e33196fc5ae1e0df363">abd9ffc838dbd33170ab0e33196fc5ae1e0df363</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-7.liferay.com/job/test-portal-source-format/2475/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-7.liferay.com/userContent/jobs/test-portal-source-format/builds/2475/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 18 out of 18 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 34 out of 34 jobs passed in 1 hour 9 minutes 40 seconds 907 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/06fc842158775ba2b96fcd2e9740b8cbd5137de1">06fc842158775ba2b96fcd2e9740b8cbd5137de1</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/b4536024712dac54820c4c13f7cf737b2b7a945b">b4536024712dac54820c4c13f7cf737b2b7a945b</a></p><details><summary><strong>ci:test:stable - 18 out of 18 jobs PASSED</strong></summary><details><summary><strong>18 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/382638/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/348943/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/378855/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403485/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/382443/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/382444/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/387686/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/359100/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386582/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326758/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/359099/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/348222/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/387687/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/382640/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/382445/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386581/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/314815/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252830/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 34 out of 34 jobs PASSED</strong></summary><details><summary><strong>34 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest(master)/5707/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252828/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403487/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/382638/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/312304/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395473/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252829/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326757/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/348943/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/378855/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403485/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/382443/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/382444/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/387686/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/359100/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386582/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326758/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/359099/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/348222/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/387687/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/348221/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/382639/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395470/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/378856/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395474/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/382640/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395471/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403486/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/382445/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386581/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/314815/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395472/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/314814/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252830/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable</a></li></ul></details></details><h5>For more details click <a href="https://test-1-4.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5707/jenkins-report.html">here</a>.</h5></details></html>

oliveiraaraujo commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
Skipping previously passed test suites:
`ci:test:relevant`
`ci:test:sf`

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pullrequest to `brianchandotcom`.

liferay-continuous-integration commented:
Pull request has been successfully forwarded to https://github.com/brianchandotcom/liferay-portal/pull/87851
